bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -14,6 +15,11 @@ PlotCanvas = require("./plot_canvas").Model
 class PlotView extends LayoutDOM.View
   className: "bk-plot-layout"
 
+  bind_bokeh_events: () ->
+    super()
+    title_msg = "Title object cannot be replaced. Try changing properties on title to update it after initialization."
+    @listenTo(@model, 'change:title', () => logger.warn(title_msg))
```

Comment:
Yes, this is temporary, and may be resolved by #4459 if it can get in in time, I need to try re-doing that PR on top of yours, though.